models/migrations/v1_25/v322.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_25
+
+import (
+	"code.gitea.io/gitea/modules/timeutil"
+
+	"xorm.io/xorm"
+)
+
+func AddUserSSHKeypairTable(x *xorm.Engine) error {
+	type UserSSHKeypair struct {
```

Comment:
The existing modules (including: activitypub, rpm/alpine/arch/debian package registry) all use user_setting table to store the key pairs.

Reply:
Thanks, I've updated this PR to use the user_setting table per your suggestion.